src/main/java/com/crowd/service/ProjectService.java

```diff
@@ -125,7 +125,9 @@ public void checkProjectStatus(Project project) {
 
     public ProjectResponse getProjectById(int projectId){
         ProjectResponse projectResponse = new ProjectResponse(projectDAO.findById(projectId));
-        projectResponse.isEditable(authService.isSameWithAuthUser(userDAO.findById(projectResponse.getUserId())));
+        boolean isSameUser = authService.isSameWithAuthUser(userDAO.findById(projectResponse.getUserId()));
```

Comment:
rename to `isAuthenticated` - this name better represents it's responsibility in context of `authService`.